<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>856</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <weight>50</weight>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>BOT FOR COD</string>
  </property>
  <property name="toolTip">
   <string/>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="locale">
   <locale language="Polish" country="Poland"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <widget class="QCheckBox" name="checkBox_goldmap">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>150</y>
      <width>175</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>Złoto</string>
    </property>
    <property name="shortcut">
     <string/>
    </property>
    <property name="autoRepeat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_woodmap">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>190</y>
      <width>175</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>Drewno</string>
    </property>
    <property name="shortcut">
     <string/>
    </property>
    <property name="autoRepeat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_stonemap">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>230</y>
      <width>175</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>Kamień</string>
    </property>
    <property name="shortcut">
     <string/>
    </property>
    <property name="autoRepeat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_manamap">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>270</y>
      <width>175</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>Mana</string>
    </property>
    <property name="shortcut">
     <string/>
    </property>
    <property name="autoRepeat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_start">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>180</width>
      <height>42</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Uruchom bota</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_stop">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>20</y>
      <width>180</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Zatrzymaj bota</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_config">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>20</y>
      <width>180</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Kofiguruj położenie budynków</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_logs">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>10</y>
      <width>301</width>
      <height>831</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_research">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>190</y>
      <width>175</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>Badania</string>
    </property>
    <property name="shortcut">
     <string/>
    </property>
    <property name="autoRepeat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_heal">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>230</y>
      <width>175</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>Leczenie jednostek</string>
    </property>
    <property name="shortcut">
     <string/>
    </property>
    <property name="autoRepeat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_autobuild">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>150</y>
      <width>175</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>Budowa (auto)</string>
    </property>
    <property name="shortcut">
     <string/>
    </property>
    <property name="autoRepeat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_1">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>270</y>
      <width>175</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="shortcut">
     <string/>
    </property>
    <property name="autoRepeat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>80</y>
      <width>190</width>
      <height>270</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string>Szkolenie legionów</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignHCenter|Qt::AlignTop</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_autobuild_2">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>150</y>
      <width>175</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>Budowa (auto)</string>
    </property>
    <property name="shortcut">
     <string/>
    </property>
    <property name="autoRepeat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_autobuild_3">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>190</y>
      <width>175</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>Budowa (auto)</string>
    </property>
    <property name="shortcut">
     <string/>
    </property>
    <property name="autoRepeat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_autobuild_4">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>230</y>
      <width>175</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>Budowa (auto)</string>
    </property>
    <property name="shortcut">
     <string/>
    </property>
    <property name="autoRepeat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_autobuild_5">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>270</y>
      <width>175</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>Budowa (auto)</string>
    </property>
    <property name="shortcut">
     <string/>
    </property>
    <property name="autoRepeat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_autobuild_6">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>310</y>
      <width>175</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>Budowa (auto)</string>
    </property>
    <property name="shortcut">
     <string/>
    </property>
    <property name="autoRepeat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_delay">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>10</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_restart">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>50</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>80</y>
      <width>190</width>
      <height>230</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string>W mieście</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignHCenter|Qt::AlignTop</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>190</width>
      <height>230</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string>Zbieraj surowce na mapie</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignHCenter|Qt::AlignTop</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <zorder>checkBox_woodmap</zorder>
   <zorder>checkBox_manamap</zorder>
   <zorder>checkBox_goldmap</zorder>
   <zorder>checkBox_stonemap</zorder>
   <zorder>pushButton_start</zorder>
   <zorder>pushButton_stop</zorder>
   <zorder>pushButton_config</zorder>
   <zorder>textEdit_logs</zorder>
   <zorder>checkBox_research</zorder>
   <zorder>checkBox_heal</zorder>
   <zorder>checkBox_autobuild</zorder>
   <zorder>checkBox_1</zorder>
   <zorder>label_3</zorder>
   <zorder>checkBox_autobuild_2</zorder>
   <zorder>checkBox_autobuild_3</zorder>
   <zorder>checkBox_autobuild_4</zorder>
   <zorder>checkBox_autobuild_5</zorder>
   <zorder>checkBox_autobuild_6</zorder>
   <zorder>lineEdit_delay</zorder>
   <zorder>lineEdit_restart</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
